languages/swift/exercises/concept/strings-and-characters/.docs/introduction.md

```diff
@@ -0,0 +1,111 @@
+
+```swift
+let aString = "A"
+// Swift infers this to be a String
+
+var aChar: Character = "A"
+// Swift now knows to make this a Character
+
+let badChar: Character = "Too many characters"
+// Error: Cannot convert value of type 'String' to specified type 'Character'
+```
+
+Like other constructs in Swift, characters and strings are immutable if defined using `let` and mutable if defined using `var`. So in the above examples, `hello` and `sausage` are immutable strings while `fam` and `empty`, AND `aChar` are mutable.
+
+### Building strings from smaller parts
+
+Strings can be concatenated using the `+` operator, and mutable strings can have other strings appended onto them using the `+=` mutating operator.
+
+```swift
+"honey" + "comb"
+// => "honeycomb"
+
+fam += ", this is our cat: 🐱"
+// => "This is my family: 👨‍👩‍👦‍👦, this is our dog: 🐶, this is our cat: 🐱"
+```
+
+Note that only strings can be concatenated with strings. In order to use concatenation with a character, one must convert it to a String first
+
+```swift
+let question: Character = "?"
+let areYouHappy = "Happy" + question
+// Error: Cannot convert value of type 'Character' to expected argument type 'String'
+
+let areYouHappy = "Happy" + String(question)
+// => "Happy?"
+```
+
+### String Interpolation
+
+The most common way to build up strings in Swift is _string interpolation_. To insert a value into a string using string interpolation, you place an expression or value in parentheses which are preceded by a backslash character.
+
+```swift
+let radius = 5.0
+let interp = "The area of a circle with radius \(radius) is \(Double.pi * radius * radius)"
+// => "The area of a circle with radius 5.0 is 78.53981633974483"
+```
+
+### String and Character equality
+
+Strings and Characters can be compared for equality (or lack thereof) using the `==` operator (or `!=` for not-equal).
+
+```swift
+let hi = "Hello"
+let period: Character = "."
+"\(hi)\(period)" == "Hello."
+// => true
+
+period != ";"
+// => true
+```
+
+### String and Character properties
+
+Strings and characters have a few different _properties_ which can be queried to get information about the string or character. These are queried by placing a ._propertyName_ after the string or character in question For example, one can check to see if a string is empty by querying it's `isEmpty` property, and the count of characters in a string can be retrieved using its `count` property.
+
+Character properties are mostly based on the type of character it is. For example, one can check to see if a character `isNumber`, `isLetter`, `isUppercase`, etc.
+
+```swift
+empty.isEmpty
+// => true
+fam.count
+// => 60
+aChar.isLowercase
+// => false
+let half: Character = "½"
+half.isNumber
+// => true
+```
```

Comment:
I like the beginning of this explanation. I would suggest to make make the text be a bit shorter, e.g. by using less examples. The general point that you want to get across to students is that there are such things as properties, that you can access those using dot-notation and that strings and characters have properties. 

The after.md document can be a bit more expansive though.